QualityControl/docs/Architecture.md

```diff
@@ -0,0 +1,26 @@
+# Project Architecture
+This document explains the architecture of the project, detailing the main components and their interactions.
+
+## Overview
+The project is a web-based graphical user interface (GUI) for O<sup>2</sup> Quality Control. It consists of a frontend built with Mithril.js and a backend built with Node.js and Express.js. The backend interacts with the Calibration and Conditions Database (CCDB) to retrieve and manage quality control data.
```

Comment:
I believe some redirect links to those technologies might be good here: NodeJS, ExpressJS, CCDB